package.json

```diff
@@ -25,7 +25,7 @@
     "await-poll": "^1.0.2",
     "bn.js": "^5.1.3",
     "buffer": "^6.0.2",
-    "class-transformer": "^0.3.1",
+    "class-transformer": "^0.3.2",
```

Comment:
I figured out that the problem I had after the framework update was caused by a different version of `class-transformer`. Could you please change this to strict version number `0.3.2`?